### PR TITLE
HRIN-533: Added notification to page title

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/HRIN-533

### CODE REVIEW

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module [hunk 1]
```diff
@@ -44,3 +46,37 @@ function hoeringsportal_hearing_form_alter(&$form, FormStateInterface $form_stat
...
+      'full',
+      'hearing_ticket_add',
+      'hearing_ticket_view',
+
+    ];
+    $content_state = $entity->field_content_state->value;
+    if (in_array($view_mode, $view_mode_change_title)) {
+      switch ($content_state) {
+        case 'upcoming':
+          $start_date = $entity->field_start_date->date->format('d/m/Y');
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module:65]: I think we should use `format_date` here.

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module [hunk 2]
```diff
@@ -44,3 +46,37 @@ function hoeringsportal_hearing_form_alter(&$form, FormStateInterface $form_stat
...
+          $state = ' (' . t('Hearing deadline:') . ' ' . $start_date . ')';
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module:66]: I think we should use `t('Hearing deadline: @deadline', ['@deadline' => $start_date])` here.

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module (hunk 1 above)
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module:65]: Why are we using `field_start_date` here?

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module [hunk 3]
```diff
@@ -44,3 +46,37 @@ function hoeringsportal_hearing_form_alter(&$form, FormStateInterface $form_stat
...
+
+        case 'finished':
+          $state = ' (' . t('Hearing deadline has passed') . ')';
+          break;
+
+        default:
+          $state = '';
+      }
+      $title = $build['title'][0]['#context']['value'];
+      $new_title = t('@title @state', ['@title' => $title, '@state' => $state]);
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module:77]: I don't see any reason for using `t` to concat two strings. Did you mean to wrap `state` in `()` here, e.g. `t('@title (@state)')`?

File: web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module [hunk 4]
```diff
@@ -308,3 +310,30 @@ function _get_cancel_link(NodeInterface $node) {
...
+      if ($content_state == 'finished') {
+        $state = ' (' . t('Hearing deadline has passed') . ')';
+      }
+
+      if ($cancelled) {
+        $state = ' (' . t('Public meeting cancelled') . ')';
+      }
+
+      $title = $build['title'][0]['#context']['value'];
+      $new_title = t('@title @state', ['@title' => $title, '@state' => $state]);
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module:335]: See comments on `hoeringsportal_hearing.module`

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module (hunk 1 above)
Comment (martinyde) [reply web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module:65]: That was a feature request from designer

Comment (martinyde) [reply web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module]: Changed to  \Drupal::service('date.formatter')->format()

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module (hunk 2 above)
Comment (martinyde) [reply web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module]: Changed

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module (hunk 3 above)
Comment (martinyde) [reply web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module]: Good poitn t() removed

File: web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module (hunk 4 above)
Comment (martinyde) [reply web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module]: Good point t() removed

File: web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module [hunk 5]
```diff
@@ -44,3 +46,37 @@ function hoeringsportal_hearing_form_alter(&$form, FormStateInterface $form_stat
...
+      'full',
+      'hearing_ticket_add',
+      'hearing_ticket_view',
+
+    ];
+    $content_state = $entity->field_content_state->value;
+    if (in_array($view_mode, $view_mode_change_title)) {
+      switch ($content_state) {
+        case 'upcoming':
+          $start_date = \Drupal::service('date.formatter')->format($entity->field_start_date->date->getTimestamp(), 'hoeringsportal_date');
```
Comment (rimi-itk) [inline web/modules/custom/hoeringsportal_hearing/hoeringsportal_hearing.module:65]: Why are we using `field_start_date` here? We should use the deadline, right?

**Pull request merged**